test/config/CRP4-session.data.xml

```diff
@@ -83,8 +83,49 @@
  <comment creation-time="20240307T133917" created-by="glehmann" created-on="np04-srv-024.cern.ch" author="Unknown" text="disable TPG"/>
  <comment creation-time="20240426T073141" created-by="thea" created-on="np02-srv-001.cern.ch" author="thea" text="enabling callbacks"/>
  <comment creation-time="20240430T135210" created-by="thea" created-on="np02-srv-001.cern.ch" author="thea" text="Removed ta source id"/>
+ <comment creation-time="20240613T081813" created-by="rsipos" created-on="np04-srv-031.cern.ch" author="rsipos" text="wib app"/>
+ <comment creation-time="20240613T084542" created-by="rsipos" created-on="np04-srv-031.cern.ch" author="rsipos" text="wib modules"/>
+ <comment creation-time="20240613T084730" created-by="rsipos" created-on="np04-srv-031.cern.ch" author="rsipos" text="wib app"/>
+ <comment creation-time="20240613T123312" created-by="rsipos" created-on="np04-srv-004.cern.ch" author="rsipos" text="cold adc defaults"/>
+ <comment creation-time="20240613T135746" created-by="rsipos" created-on="np02-srv-004.cern.ch" author="rsipos" text="coldadc and pulser settings"/>
 </comments>
 
+<obj class="ColdADCSettings" id="def-coldadc-settings">
+ <attr name="reg_0" type="u8" val="35"/>
+ <attr name="reg_4" type="u8" val="59"/>
+ <attr name="reg_24" type="u8" val="223"/>
+ <attr name="reg_25" type="u8" val="51"/>
+ <attr name="reg_26" type="u8" val="137"/>
+ <attr name="reg_27" type="u8" val="103"/>
+ <attr name="reg_29" type="u8" val="39"/>
+ <attr name="reg_30" type="u8" val="39"/>
+</obj>
+
+<obj class="DaqApplication" id="hermes-CRP4">
```

Comment:
There is a WIECApplication called hermes-CRP4 in this file. I don't think we can have 2 applications with the same name. Although we can't yet test the WIECApplication as the module generating code doesn't exist yet, one should be removed or renamed.

Reply:
Yes, this is a residual object that I wanted to keep for the same behavior. I didn't hear from the WIECApplication since today afternoon. I'll remove this offending app.